modules/rpc/yarpc_test.go

```diff
@@ -36,30 +35,36 @@ func TestRegisterDispatcher_OK(t *testing.T) {
 	RegisterDispatcher(defaultYARPCDispatcher)
 }
 
+func TestRegisterStarter_OK(t *testing.T) {
+	t.Parallel()
+	RegisterStarter(defaultYARPCStarter)
```

Comment:
You are right about this possibility and that's why I have only a defaultStarter.